superset/db_engine_specs/base.py

```diff
@@ -206,6 +206,10 @@ def is_db_column_type_match(
     def get_allow_cost_estimate(cls, version: Optional[str] = None) -> bool:
         return False
 
+    @classmethod
```

Comment:
This check blocks requests to those databases that don't support stop queries function, or those companies that does not have stop queries API.

Reply:
Alternatively, if `stop_queries` was an empty function, you might be able to remove the `allows_stop_pending_queries` attribute altogether. I think the only reason we might need the database level allows attribute would be if you wanted to prevent the frontend from sending the request to stop queries. But that could be messy as dashboards can be based off multiple different databases.

So maybe remove the `allows` check completely and just make `stop_queries` an empty function here

Reply:
+1 to this suggestion

Reply:
thanks for the suggestion. fixed by remove `get_allow_stop_pending_queries` check, and add an empty `stop_queries`.